geode-core/src/integrationTest/java/org/apache/geode/distributed/internal/membership/gms/messenger/JGroupsMessengerJUnitTest.java

```diff
@@ -430,11 +426,8 @@ public void testSendUnreliably() throws Exception {
       when(msg.getMulticast()).thenReturn(enableMcast);
       when(msg.getDSFID()).thenReturn((int) DataSerializableFixedID.HEARTBEAT_RESPONSE);
       interceptor.collectMessages = true;
-      try {
-        messenger.sendUnreliably(msg);
-      } catch (GemFireIOException e) {
-        fail("expected success");
-      }
+      Set<?> failures = messenger.sendUnreliably(msg);
+      assertTrue(failures == null || failures.isEmpty());
```

Comment:
This will be a lot easier to debug if it fails using `assertThat(failures).isEmpty(). Otherwise we never see the actual failures.